.github/workflows/linux_meson.yml

```diff
@@ -26,7 +22,7 @@ permissions:
 
 jobs:
   meson_devpy:
-    if: "github.repository == 'numpy/numpy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
+    if: "github.repository == 'numpy/numpy'"
```

Comment:
@rgommers: why did you remove the conditions from this stanza? Is there a replacement mechanism?

Reply:
This has had builtin support for quite a while now, see https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs. So we shouldn't continue replication this boilerplate. I can open a separate PR to remove it from the other jobs now. Sound good?

Reply:
Ahh, cool. It makes sense to remove this from the other github workflows then.